AUTHORS

```diff
@@ -451,3 +451,4 @@ Dmitry Litvinchenko <karaliti@gmail.com>
 chocolateboy <chocolate@cpan.org>
 Henry Zhu <hi@henryzoo.com>
 Nate Goldman <ungoldman@gmail.com>
+Ted Yavuzkurt <hello@TedY.io>
```

Comment:
AUTHORS is machine generated (when we do releases). To use a name/email address other than the one on your git commits we add an entry to `.mailcap`.